balena/auth.py

```diff
@@ -125,7 +125,6 @@ def who_am_i(self):
 
         Examples:
             >>> balena.auth.who_am_i()
-            u'g_trong_nghia_nguyen'
```

Comment:
Yeah, overall I would like to remove the examples from the documentations as everytime that api does some change they get obsolate and not necessarily the SDK tests will run when API changes so it makes it very hard to mantain. My bet here is that it will be a better developer experience and easier to mantain (as we can basically just map changes on node-sdk types) if we replace it by type hints.

Reply:
What we might want to look into in the future (for both node and python sdks) is generating the examples from a job that runs when balena-api/open-balena-api is merged. For example, the outputs from e2e tests or something. This would be much more valuable imho